tensorflow/tools/pip_package/build_pip_package.sh

```diff
@@ -107,7 +107,7 @@ function main() {
   mkdir -p ${TMPDIR}/third_party
   pushd ${RUNFILES%org_tensorflow}
   for header in $(find protobuf -name \*.h); do
-    cp --parents "$header" ${TMPDIR}/google;
+    rsync -R "$header" ${TMPDIR}/google;
```

Comment:
Just noticed that there is a variable called "dest_dir" in the same file. So "DEST_DIR" should be renamed to something else to avoid confusion.